### PR TITLE
Add examples for using Markers

### PR DESCRIPTION
**Pull request by rahsheen**
Ran into someone struggling with this and also noticed there's issue #14  so attempted to fill this gap with a few examples based on the official docs.

### CODE REVIEW

**Review by Bill0197 — APPROVED** [review]

**Comment by rahsheen** [comment]
Any reason this cannot be merged?

File: markers-custom/src/Map.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  const markerClicked = (title) => {
+    window.alert(title);
+  };
+
+  return (
+    <div>
+      <div className="sidebarStyle">
+        <div>
+          Longitude: {lng} | Latitude: {lat} | Zoom: {zoom}
+        </div>
```
Comment (tristen) [inline markers-custom/src/Map.js]: For simplicity, can you drop this feature and logic above from your marker examples as it's not related?

File: markers-custom/src/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Map from './Map';
+
+function App() {
+  return (
+    <div>
+      <Map />
+    </div>
```
Comment (tristen) [inline markers-custom/src/App.js]: Nit picky but this probably doesn't need to be wrapped in div tags?

File: markers-custom/src/Map.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    // Clean up on unmount
+    return () => map.remove();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const markerClicked = (title) => {
+    window.alert(title);
+  };
+
+  return (
+    <div>
```
Comment (tristen) [inline markers-custom/src/Map.js]: Intead of an extra wrapping div, could you use empty tag [framents](https://reactjs.org/docs/fragments.html#short-syntax)?

File: markers-custom/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    git clone https://github.com/mapbox/mapbox-react-examples.git
+
+    cd mapbox-react-examples/basic
+    npm install
+    npm start
+
+    open http://localhost:3000/
+
+Runs the app from http://localhost:3000
```
Comment (tristen) [inline markers-custom/README.md]: Can you update the READMEs of the three examples to reflect what they do? No need to include a visual example and these can be pretty short.

File: markers-custom/package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "basic",
```
Comment (tristen) [inline markers-custom/package.json]: Can you update the name field here to the directory name of your examples?

**Review by tristen — CHANGES_REQUESTED** [review]
Thanks for the PR! These look great. I just left a few comments on a couple of things I saw that should be updated before merging.

**Comment by rahsheen** [comment]
Sounds good. Implemented the suggestions. Thanks!

**Review by tristen — APPROVED** [review]
Looks great!

**Pull request merged**